copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>What does the result of printing show?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>The execute command shows the count of number of points. Added a comment to highlight this in the notebook.</p>